templates/docs/examples/patterns/contextual-menu/default.html

```diff
@@ -1,7 +1,7 @@
 {% extends "_layouts/examples.html" %}
 {% block title %}Contextual menu / Default{% endblock %}
 
-{% block standalone_css %}patterns_contextual-menu{% endblock %}
+{% block standalone_css %}patterns_code-snippet{% endblock %}
```

Comment:
Why the change in contextual menu example?

Reply:
too many editor tabs open...